\doxysection{pandas.\+errors.\+Attribute\+Conflict\+Warning Class Reference}
\hypertarget{classpandas_1_1errors_1_1AttributeConflictWarning}{}\label{classpandas_1_1errors_1_1AttributeConflictWarning}\index{pandas.errors.AttributeConflictWarning@{pandas.errors.AttributeConflictWarning}}


Inheritance diagram for pandas.\+errors.\+Attribute\+Conflict\+Warning\+:
% FIG 0


Collaboration diagram for pandas.\+errors.\+Attribute\+Conflict\+Warning\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Warning raised when index attributes conflict when using HDFStore.

Occurs when attempting to append an index with a different
name than the existing index on an HDFStore or attempting to append an index with a
different frequency than the existing index on an HDFStore.

Examples
--------
>>> idx1 = pd.Index(['a', 'b'], name='name1')
>>> df1 = pd.DataFrame([[1, 2], [3, 4]], index=idx1)
>>> df1.to_hdf('file', 'data', 'w', append=True)  # doctest: +SKIP
>>> idx2 = pd.Index(['c', 'd'], name='name2')
>>> df2 = pd.DataFrame([[5, 6], [7, 8]], index=idx2)
>>> df2.to_hdf('file', 'data', 'a', append=True)  # doctest: +SKIP
AttributeConflictWarning: the [index_name] attribute of the existing index is
[name1] which conflicts with the new [name2]...
\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{pandas_2errors_2____init_____8py_source_l00693}{693}} of file \mbox{\hyperlink{pandas_2errors_2____init_____8py_source}{\+\_\+\+\_\+init\+\_\+\+\_\+.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/pandas/errors/\+\_\+\+\_\+init\+\_\+\+\_\+.\+py\end{DoxyCompactItemize}
